frontend/src/app/core/models/generalAvailabilitySettings.ts

```diff
@@ -0,0 +1,21 @@
+export interface GeneralAvailabilitySettings {
```

Comment:
```suggestion
export interface IGeneralAvailabilitySettings {
```

Reply:
Should the file be named IGeneral... or something along those lines or should I leave it as it is?

Reply:
It would be better to change name as well